resources/js/Pages/Home.vue

```diff
@@ -12,6 +12,7 @@
                 :label="$i18n('item-form-id-input-label')"
                 :placeholder="$i18n('item-form-id-input-placeholder')"
                 :rows="8"
+                :error="error ? {message: $i18n(error.message), type: error.type} : null"
```

Comment:
I think we can simplify this a little, by moving the call to the `$i18n` function into the validation method.

The goal would be to be left with:

```suggestion
                :error="error"
```